data-model/src/main/scala/za/co/absa/enceladus/model/Validation.scala

```diff
@@ -27,24 +27,44 @@ object Validation {
 
 }
 
-case class Validation (errors: Map[String, List[String]] = Map()) {
```

Comment:
**Minor suggestion:** `Map[String, List[String]]` could have be defined as a `type` in the companion object. Would shorten, simplify and perhaps make it easier to understand the code.

(Placed the comment on original code, as GitHub doesn't allow me to place it to the changed one for unknown reason.)